src/app/Http/Controllers/FinancialOperations/OperationsOverviewController.php

```diff
@@ -22,17 +22,7 @@ class OperationsOverviewController extends Controller
      * @var int
      * number of operations to be shown on one page
      */
-    public static int $perPage = 15;
-
-    /**
-     * shortcut for the $perPage static variable
-     *
-     * @return int
-     */
-    private function getPerPage()
-    {
-        return OperationsOverviewController::$perPage;
-    }
+    public static int $resultsPerPage = 15;
```

Comment:
```suggestion
    private static int $resultsPerPage = 15;
```

Use HiddenMembersAccessor if you need this during testing.